examples/cosmos/ics20/ics20.qnt

```diff
@@ -45,10 +45,12 @@ module ics20 {
   type ChainState = {
     bank: Accounts,
     channels: ChannelCounterparties,
-    outPackets: Set[Packet],
-    receivedButUnacknowledgedPackets: Set[Packet],
-    inAcknowledgements: Set[FungibleTokenPacketAcknowledgement],
     channelEscrowAddresses: Channel -> Address,
+    // IBC packet handling:
+    outPackets: Set[Packet],                        // outgoing unreceived packets
```

Comment:
what does it mean to be outgoing and unreceived? I guess, a sender chain would not immediately learn that its packet has been received, until the moment it receives an acknowledgment. Would something like `sentNoAckPackets` describe this set better?

Reply:
> Would something like `sentNoAckPackets` describe this set better?

I don't see how this helps, it's really receival that's decisive here, because producing and consuming the ack is asynchronous.

Reply:
We could probably separate this out into a separate `RelayerState`. However, it's not obvious where to draw the line between on-chain state and relayer state, where they overlap.

Reply:
We have a separate issue for addressing this, so we can follow up there: https://github.com/informalsystems/apalache-team-services/issues/7